docs/source/rest_api/endpoint_specs.rst

```diff
@@ -0,0 +1,10 @@
+***********************
+Endpoint Specifications
+***********************
+
+These specifications document the available Sawtooth REST API endpoints. They are generated from the official `OpenAPI <http://swagger.io/specification/>`_ formatted YAML specification, which can be found on the `Sawtooth Lake Github <https://github.com/hyperledger/sawtooth-core/blob/master/rest_api/sawtooth_rest_api/openapi.yaml>`_.
+
+.. note::
+   This specification has not been fully implemented. Some query parameters may not function or behave as specified.
```

Comment:
Do we have a way to mark which have not been implemented?  It seems like the complete spec is the design, but the doc should only be what's there (e.g. we might decide not to implement any of the unimplemented params until v2)